tools/invoke-tests

```diff
@@ -36,6 +36,9 @@ while [[ $# -gt 0 ]]; do
             ;;
         --prove-tool)
             prove_path=$2
+            echo "-----------------------------"
+            echo "Using prove tool: $prove_path"
+            echo "-----------------------------"
```

Comment:
This can probably be removed again and CMake logs this anyway via ` 3: Test command: /opt/tools/invoke-tests "--prove-tool" "/opt/tools/prove_wrapper" "--make-tool" "/usr/bin/ninja" "--unbuffer-tool" "UNBUFFER_PATH-NOTFOUND" "--build-directory" "/opt/build"`.

Reply:
Yep, I didn't want to be permanent, but I was using only for debugging purpose :+1: